weblab/core/processing.py

```diff
@@ -0,0 +1,11 @@
+from urllib.parse import urljoin
+
+from django.conf import settings
+
+def prepend_callback_base(url_path):
+    """Prepend CALLBACK_BASE_UR to url_path. If we are running in a subfolder this is striped from url_path."""
+
+    if hasattr(settings, 'FORCE_SCRIPT_NAME'):
+         url_path = url_path.replace(settings.FORCE_SCRIPT_NAME, '')
```

Comment:
Just in case someone chooses a prefix that also appears later in some URLs, I think I'd do this instead:
```suggestion
    if hasattr(settings, 'FORCE_SCRIPT_NAME') and url_path.startswith(settings.FORCE_SCRIPT_NAME):
         url_path = url_path[len(settings.FORCE_SCRIPT_NAME):]
```

Reply:
I think this should od it too?:
```
url_path = url_path.replace(settings.FORCE_SCRIPT_NAME, '', `)
```